autospec/pkg_integrity.py

```diff
@@ -244,8 +244,8 @@ def get_signature_file(package_url, package_path):
                 return sign_file
             elif os.path.exists(dest):
                 os.unlink(dest)
-        except pycurl.error as e:
-            print(e.args)
+        except pycurl.error:
+            print(f"Download of signature file {url} failed")
```

Comment:
Do we care to print the HTTP status code here?

Reply:
I didn't think it really mattered as folks who are confused as to why a URL didn't work probably try it anyway, most of the time failure is expected as it is just guessed URLs being tried.

Reply:
Okay, just checking, since printing `e.args` did report the status code :) I don't have strong opinions either way.